zulipterminal/ui_tools/views.py

```diff
@@ -302,6 +302,10 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             self.log.set_focus(self.focus_index_before_search)
             self.view.controller.update_screen()
             return key
+        elif is_command_key('GO_UP', key):
+            key = 'up'
+        elif is_command_key('GO_DOWN', key):
+            key = 'down'
```

Comment:
One aspect which makes me wonder if we should approach this differently is that scrolling with the extra keys (vim-like) already works in the stream-view, from the base `View`? So it would be simpler if we could avoid adding extra code and tests just for this case.

I can follow why you've added this, but this and the associated test are separate code which are needed before the actual mouse_event is changed - so if we do take this route, I think we should put this in a separate commit, at least for now.

Reply:
> I had to add GO_UP/DOWN presses for Streams/Topics/UsersView views. They already support navigation through alternative navigation keys (this is defined in View class) but since mouse_event directly calls its keypress method, the addition was necessary to make it work. 

I think it is necessary to add these to make this work. I have separated these changes in another commit 381908c5f1a1fa5898a46017900fe2a66cd3e327 with their tests.